### PR TITLE
Feature/add name to form

### PR DESCRIPTION
**Pull request by sagarbhandari-0522**
We need to add first-name and last name too in the registration form as per the client request

### CODE REVIEW

File: djangoAssignment1/settings.py [hunk 1]
```diff
@@ -9,13 +9,13 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/5.0/ref/settings/
 """
-from datetime import datetime
+import secrets
+from datetime import datetime, timedelta
```
Comment (shashankdhungel) [inline djangoAssignment1/settings.py:13]: why you didn't use django datetime

**Review by shashankdhungel — CHANGES_REQUESTED** [review]
see comments please

File: djangoAssignment1/settings.py (hunk 1 above)
Comment (sagarbhandari-0522) [reply djangoAssignment1/settings.py:13]: Just leave it as it is there please

**Comment by shashankdhungel** [comment]
- [x] Integration testing
- [x] Build
- [x] commits
- [x] changes

**Comment by shashankdhungel** [comment]
Please could you re-write your descriptions on which part of the project we work on?

**Pull request closed without merging**